frontend/src/__tests__/cypress/cypress/tests/mocked/projects/tabs/permissions.cy.ts

```diff
@@ -172,6 +182,60 @@ describe('Permissions tab', () => {
 
       cy.wait('@deleteUser');
     });
+
+    it('Shows confirmation modal when editing own permissions', () => {
+      initIntercepts({ isEmpty: false });
+      permissions.visit('test-project');
+
+      userTable.getTableRow('test-user').findKebabAction('Edit').click();
+      userTable.findEditInput('test-user').clear().type('test-user');
+      userTable.selectPermission('test-user', 'Admin Edit the project and manage user access');
+      userTable.findEditSaveButton('test-user').click();
+
+      permissions.findConfirmModal().should('be.visible');
+      permissions.findModalCancelButton().click();
```

Comment:
you should be able to import the base modal and just use it's cancel and submit buttons. Can also delete the `find` functions you made above if you do that